packages/@expo/cli/README.md

```diff
@@ -126,11 +132,13 @@ There are two testing scripts:
 ### E2E Testing Guidelines
 
 - E2E tests should be resilient and reliable, be sure to give them plenty of time for network requests.
-- When testing locally you should attempt to reuse node modules for faster results. In the `npx expo prebuild` and `npx expo start` commands for instance, we utilize a helper method that will default to reusing a project + node_modules when run locally. This can be [toggled off](https://github.com/expo/expo/blob/11a5a4d27b7e1c8e4d6ddf0401397d789d89f52a/packages/%40expo/cli/e2e/__tests__/utils.ts#L174) to bootstrap a fresh project every time.
+- When testing locally you had to have Bun installed, it can be done via Volta - `volta install bun`.
+- When testing locally you should attempt to reuse node modules for faster results. In the `npx expo prebuild` and `npx expo start` commands for instance, we utilize a helper method that will default to reusing a project + **node_modules** when run locally. This can be [toggled off](https://github.com/expo/expo/blob/11a5a4d27b7e1c8e4d6ddf0401397d789d89f52a/packages/%40expo/cli/e2e/__tests__/utils.ts#L174) to bootstrap a fresh project every time.
 - When bootstrapping test projects, utilize the temporary folder `os.tmpdir()` as this folder is automatically cleaned up when the computer restarts.
 
 ## Coming from Expo CLI
 
+> [!important]
```

Comment:
I don't think it's that important tbh 😄 It's great for context, no one should be on `expo-cli` anymore.